runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOreOverlay.java

```diff
@@ -65,60 +71,68 @@ public Dimension render(Graphics2D graphics)
 		int runiteFound = session.getRuniteFound();
 
 		// If no ores have even been collected, don't bother showing anything
-		if (nuggetsFound == 0 && coalFound == 0 && goldFound == 0 && mithrilFound == 0
-			&& adamantiteFound == 0 && runiteFound == 0)
+		// This will statement is true iff all ore counts are 0
+		if ((nuggetsFound | coalFound | goldFound | mithrilFound | adamantiteFound | runiteFound) == 0)
```

Comment:
This doesn't need to be changed.

Reply:
Yeah was trying to be fancy and save some CPU cycles but grand scheme doesn't matter so will revert back.